research/05-problem-ana.tex

```diff
@@ -1,10 +1,127 @@
 \section{Problem Analysis}
 \label{sec:problem-analysis}
+This section gives an analysis of the problem as defined in the
+previous section \ref{sec:problem-definition}. More specifically, this
```

Comment:
yea it's just that previously you wanted me to do the opposite, so it's what I did here :P.